checks/security_policy.go

```diff
@@ -71,7 +71,9 @@ func SecurityPolicy(c *checker.CheckRequest) checker.CheckResult {
 		dotGitHub.RepoClient = dotGitHubClient
 		onFile = func(name string, dl checker.DetailLogger, data FileCbData) (bool, error) {
 			pdata := FileGetCbDataAsBoolPointer(data)
-			if strings.EqualFold(name, "security.md") {
+			if strings.EqualFold(name, "security.md") ||
+				strings.EqualFold(name, ".github/security.md") ||
```

Comment:
this seems already covered in lines 61-62.

i think apache one is in
https://github.com/apache/.github/blob/main/.github/SECURITY.md

so it is .github/.github/SECURITY.md instead of .github/SECURITY.md. that seems wrong place

Reply:
No, according to the github docs, it can be .github/.github/SECURITY.md or .github/docs/SECURITY.md or .github/SECURITY.md

i.e. https://github.com/apache/couchdb/security/policy shows this is valid.

Reply:
also, docs/security.md change should go near line 42.

Reply:
> also, docs/security.md change should go near line 42.

No, this is specifically the .github organisational security.md files.